operations/rates.md

```diff
@@ -104,6 +104,123 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
+| `Rates` | array of [Rate parameters](#rate-parameters) | required, max 1000 items | Information about rates to be created. |
+
+#### Rate parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ServiceId` | string | require | Unique identifiers of the [Services](services.md#service). |
+| `RateGroupId` | string | required | Unique identifier of the [Rate group](rates.md#rate-group) under which rate is assigned. |
+| `IsEnabled` | bool | optional | Whether the rate is available to customers. `False` will be used as a default when not provided. |
+| `Type` | [Add Rate type](rates.md#add-rate-type) | required | Type of the rate. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](accountingcategories.md#accounting-category) the rate belongs to. |
+| `BusinessSegmentId` | string | optional | Unique identifier of the [Business segment](businesssegments.md#business-segment). |
+| `Name` | [Localized text](_objects.md#localized-text) | required | Name of the rate. |
+| `ShortName` | [Localized text](_objects.md#localized-text) | optional | Short name of the rate. |
+| `ExternalNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the external name of the rate. |
+| `Descriptions` | [Localized text](_objects.md#localized-text) | optional | All translations of the description. |
+| `PricingType` | [Pricing type](rates.md#pricing-type) | required | Discriminator in which field inside `Pricing` contains additional data. |
+| `Pricing` | [Pricing](rates.md#rate-pricing-parameters) | required | Contains additional data about pricing of the rate. |
+
+### Rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRatePricing` | [Base rate pricing](rates.md#base-rate-pricing-parameters) | optional | Additional data for rate with base rate pricing. |
+| `DependentRatePricing` | [Base rate pricing](rates.md#dependent-rate-pricing-parameters) | optional | Additional data for rate with dependent rate pricing. |
+
+### Base rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Amount` | [Amount parameters](#amount-parameters) | required | Unit amount of the product that overrides the amount defined in Mews. |
+| `EmptyCapacityAdjustment` | decimal | required |  |
+| `ExtraCapacityAdjustment` | decimal | required |  |
+
+### Dependent rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRateId` | string | required |  |
+| `RelativeAdjustment` | string | required |  |
+| `AbsoluteAdjustment` | string | required |  |
+
+#### Amount parameters
```

Comment:
I don't think you need to add the definition for `Amount parameters` here, it's already defined in `orders.md` so you could just reference it with `orders.md#amount-parameters`.